pandas/core/groupby/ops.py

```diff
@@ -592,6 +604,10 @@ def _call_cython_op(
 
         result = result.T
 
+        if self.how == "rank" and self.grouper.has_dropped_na:
+            # TODO: Wouldn't need this if group_rank supported mask
```

Comment:
would this be a matter of supporting a mask arg in the libgroupby function, or would this be just for Nullable dtype?  bc i have a branch on deck that does the former

Reply:
The former - I believe rank had to be singled out here because it was the one transform that didn't support a mask arg.

Reply:
@rhshadrach mask just got added to group_rank, but commenting this out causes a few failures.  is more needed?

Reply:
This comment was incorrect; opened #46953